src/actionlib/simple_action_client.py

```diff
@@ -219,6 +219,14 @@ def cancel_goal(self):
     def stop_tracking_goal(self):
         self.gh = None
 
+    ## @brief Set timestamp delta for lost goal ack processing
+    ##
+    ## When processing goals, compare timestamp of the goal and the
+    ## current status message. If the two differ by more than the delta,
+    ## consider the goal LOST.
```

Comment:
Copy/paste docstring is definitely anti-pattern. No way someone will know to keep these in sync if one changes.